privacy_estimates/binomial_proportion.py

```diff
@@ -77,6 +104,41 @@ def compute_eps_lo_hi(count: AttackResults, delta: float, alpha: float, method:
     return eps_lo, eps_hi
 
 
+def compute_delta_lo_hi(count: AttackResults, epsilon: float, alpha: float, method: str) -> Tuple[float, float]:
+    """
+    Computes a confidence interval for delta from equal-tailed confidence intervals for the false positive and
+    false negative rates of a membership inference attack, based on 100*(1 - alpha)% confidence intervals for epsilon
```

Comment:
I don't understand this part:  `based on 100*(1 - alpha)% confidence intervals for epsilon`. This is a standalone function, not based on `compute_eps_lo_hi`.

Reply:
Yes, but it is not clear to me what confidence intervals we have for delta, given a particular confidence interval for epsilon. They are not linearly related. Perhaps we can take this offline?

Reply:
In the abstract, $\delta$ and $\varepsilon$ are just two parameters defining a convex region in FNR-FPR space. One can fix one and estimate the other. I'm not sure what why we would want to estimate $\delta$ given an estimate of $\varepsilon$; let's chat.